app/helpers/service_helper.rb

```diff
@@ -7,6 +7,33 @@ def switch_service_link(*args)
     end
   end
 
+  def service_name
+    if TradeTariffAdmin::ServiceChooser.uk?
+      'UK Integrated Online Tariff'
+    else
+      'Northern Ireland Online Tariff'
+    end
+  end
+
+  def service_region
+    TradeTariffAdmin::ServiceChooser.uk? ? 'the UK' : 'Northern Ireland'
+  end
+
+  def replace_service_tags(content)
+    content.gsub %r{\[\[SERVICE_[A-Z_]+\]\]} do |match|
+      case match
+      when '[[SERVICE_NAME]]'
```

Comment:
same comment as before, 
if we change a bit the regexp we could just compare the match with the service name

when 'SERVICE_NAME'

Reply:
I think you start needing to rely on global match data variables - `|match|` is an instance of String rather than MatchData as you'd expect

Reply:
after discussion on the PR for `frontend` its been decided to leave this as is